src/OrchardCore.Modules/OrchardCore.Workflows/Views/Activity/EditActivity.cshtml

```diff
@@ -1,7 +1,7 @@
 @model OrchardCore.Workflows.ViewModels.ActivityEditViewModel
 
 <div data-workflow-type-id="@Model.WorkflowTypeId" data-activity-id="@Model.ActivityId">
-    <h1>@RenderTitleSegments(T["Edit {0}", Model.Activity.Name.CamelFriendly()])</h1>
+    <h1>@RenderTitleSegments(T["Edit {0}", T[Model.Activity.Name.CamelFriendly()]])</h1>
```

Comment:
This one is tricky. It's not a dynamic value (a class) and it can't be solved with a switch.
I think in this case we could create a new shape that would use the activity name in the alternate. Or add a new property on the class to return a localized string.
Another option would be to have a custom file to add new pot entries with their context, such that each module could define the activity names, and your suggestion would then work fine.

Reply:
I think adding a new property on the activity is better. Like `DisplayText`